Sources/Authentication/AppAuthSession.swift

```diff
@@ -56,6 +57,7 @@ public final class AppAuthSession: LoginSession {
         )
         
         self.state = request.state
+        self.redirectURI = request.redirectURL?.absoluteString
```

Comment:
worth pulling this straight out of the configuration instead?
Then it's not optional at all!

Reply:
Good shout, will do!